michael-msrgnt/hw_1/4-homework-99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,28 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+
+
+for (var bottleNum = 99; bottleNum > 0; bottleNum--) {
+  console.log(bottleNum + " bottles of beer on the wall, " + bottleNum + " bottles of beer. Take one down and pass it around, " + bottleNum + " bottles of beer on the wall.");
+}
+*/
+
+for (bottleNum = 99; bottleNum > 0; bottleNum) {
+  console.log(bottleNum + " bottles of beer on the wall, " + bottleNum + " bottles of beer. Take one down and pass it around, " + --bottleNum + " bottles of beer on the wall.");
+
```

Comment:
Declaring a variable and not assigning anything to it is totally cool. `var drink;`
Declaring a variable and assigning something to it at the same time is also totally cool. `var drink = "tea";`
Assigning a variable without declaring it first is _madness_! `drink = 99;`

Otherwise, concise and sound logic. 👍